metadata.rb

```diff
@@ -16,7 +16,7 @@
 supports 'oracle'
 
 depends 'build-essential'
-depends 'mysql', '>= 6.0'
+depends 'mysql',               '~> 8.0.1'
```

Comment:
and did you test if it's actually necessary to use the Cookbook version above Version 8?